_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -110,6 +119,42 @@ Heading marked up with `h` element that describes the topic or purpose of its se
 <p>We are open Monday through Friday from 10 to 16</p>
 ```
 
+#### Passed example 7
+
+Heading marked up with `h` element that describes the topic or purpose of a part of its section of the content.
+
+```html
+<h1 class="target">Oranges</h1>
+<p>I really like oranges.</p>
+<p>Apples are great too, though.</p>
+```
+
+#### Passed example 8
+
+Heading marked up with `h` element that doesn't have a section of content.
+
+```html
+<h1 class="target">Oranges on sale</h1>
+```
+
+#### Passed example 9
+
+Heading marked up with `h` element where both the visible text nodes as well as the accessible name describes the content, though the two are different.
+
+```html
+<h1 class="target" aria-label="Office opening hours">Opening Hours</h1>
+<p>We are open Monday through Friday from 10 to 16</p>
+```
+
+#### Passed example 10
+
+Visible heading is made up by an image of text, that also has an accessible name. Both visible content and accessible name is descriptive.
```

Comment:
**The** visible heading is made up **of** an image of text, that also has an accessible name. Both **the** visible content and **the** accessible name describe the content.